entry/src/main/ets/pages/Screen3.ets

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import router from '@system.router';
+
+@Entry
+@Component
+struct Screen3 {
+  private PlaceHolder_Text: string = "Enter Full Description"

```

Comment:
for variable declaration use camel case instead of snake case

Reply:
Changed declaration to Camel Case